app/views/clients/_form.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= form_for @client do |f| %>
+  <%= render 'shared/error' %>
+  <div class="box form-group">
+    <label><b><%= f.label :name %></b></label>
+    <%= f.text_field :name, autofocus: true, class: 'form-control margin-bottom' %>
+  </div>
+  <div class="box form-group">
+    <label><b><%= f.label :email %></b></label>
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class: 'form-control margin-bottom' %>
+  </div>
+  <div class="box form-group">
+    <label><b><%= f.label :country %></b></label>
+    <%=  f.country_select :country , class: 'form-control margin-bottom' %>
+  </div>
+  <div class="actions submit-button">
+    <label><b> <%= f.submit "Create", class: 'btn btn-primary active' %></b></label>
```

Comment:
spacing after b tag. and no need to use b tag here.

Reply:
REmoved.